### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by Blackspell01** [schmurtzm/RFLink32-For-Sonoff-RF-Bridge#14]
It recognises some signals (weather station and rf plugs) but i can't see anything in home assistant

**Comment by danielk117** [comment]
I think this repo isn't active anymore. Try https://github.com/cpainchaud/RFLink32 instead. It has a sonoff rfbridge firmware image under releases. 

I flashed my two bridges with the other firmware a few month ago, because of stability issues. Now they are running really well.

**Comment by schmurtzm** [comment]
Yes, sorry I don't have time currently to dedicate to this project. cpainchaud now supports directly the RF-Bridge in RFlink32. Do not hesitate to join the discord if you have questions.

**Comment by Blackspell01** [comment]
Okay, I reflashed the bridge with RFLink32 now. I know this may be the wrong place to ask but Im still facing the same problem. Also I can't really find a tutorial on how to integrate the received signals in home assistant manually

**Comment by Suschibomber** [comment]
@schmurtzm 

I am from Germany and have successfully flashed my Sonoff Bridge V2.2 with Tasmota using the Direct Hack. Unfortunately I see no signals at all.

Now I wanted to try it with your RFLink variant. My English is not the best but does your variant no longer work? Where can I find the Discord for support? And how do I have to proceed to bring RFLink to my Sonoff RF Bridge according to the current state?

Thanks in advance and thanks for your great work.

**Comment by Blackspell01** [comment]
It does work but it's based on a very old version from the original rflink (r29). My devices do not work unfortunately so it's not really usable. Wouldn't recommend the sonoff bridge atm

**Comment by Suschibomber** [comment]
@Blackspell01 

Yes I now have the Sonoff RF Bridge 2.2 and would like to test it if it receives any signal at all. For this I still have two variants to choose from. ESPHome or RFLink.
But I no longer see through all the instructions :(

Would you be so kind and could tell me how I have to proceed? I only use Windows.

**Comment by Blackspell01** [comment]
As danielk177 suggested Id install RFLink from the original repo

> I think this repo isn't active anymore. Try https://github.com/cpainchaud/RFLink32 instead. It has a sonoff rfbridge firmware image under releases.

You can use any flashing tool you'd like to, Id recommend ESPHome Flasher (available for windows).

Then you have to set up the mqtt connection to receive the 433 traffic

For ESPHome you can install your config through home assistant directly

**Comment by danielk117** [comment]
@Suschibomber If you was able to flash tasmota, then just replace the tasmota.bin with https://github.com/cpainchaud/RFLink32/releases/download/nightly/sonoff_bridge-firmware.bin and flash it again. Thats it. 

In rflink you configure mqtt and than you should see any incoming signals.

**Comment by Suschibomber** [comment]
Thanks. I will try 👍🏻:)

**Comment by heisenberg2980** [comment]
> Yes, sorry I don't have time currently to dedicate to this project. cpainchaud now supports directly the RF-Bridge in RFlink32. Do not hesitate to join the discord if you have questions.

@schmurtzm maybe you could update the main README.md to mentioned this repo is not maintained anymore so people arriving here for the first time are aware and know what to do, but please keep the instructions as they are really useful to learn how to flash the Sonoff RF Bridge.
